addons/mission/functions/fnc_monitorUnits.sqf

```diff
@@ -21,9 +21,15 @@ params [["_type", 0]];
 
 switch (_type) do {
     case 0: {
-        format ["West: %1|East: %2|Indep: %3|Civ: %4|Player: %5", west countSide allUnits, east countSide allUnits, resistance countside allUnits, civilian countSide allUnits, count playableUnits]
+        private _west = west countSide allUnits - playableUnits;
+        private _east = east countSide allUnits - playableUnits;
+        private _resistance = resistance countSide allUnits - playableUnits;
+        private _civilian = civilian countside allUnits - playableUnits;
```

Comment:
```suggestion
        private _west = west countSide (allUnits - playableUnits);
        private _east = east countSide (allUnits - playableUnits);
        private _resistance = resistance countSide (allUnits - playableUnits);
        private _civilian = civilian countside (allUnits - playableUnits);
```
It might be faster to do eg. `west countSide allUnits - count playableUnits`, so it doesn't do array operations but simple numbers, please test.

Reply:
Method already used:
0.112994 ms
0.110779 ms
0.108085 ms

Your method:
0.0953 ms
0.0957 ms
0.0951 ms

Reply:
You know what to do.